src/sources/audiosource.h

```diff
@@ -67,19 +65,18 @@ class ReadableSampleFrames final: public SampleFrames {
     SampleBuffer::ReadableSlice m_readableSlice;
 };
 
-
 // Associates a range of frame indices with the corresponding
 // writable sample data as a slice in some sample buffer. The
 // memory is owned by the external sample buffer that must not
 // be modified or destroyed while writing sample data!
-class WritableSampleFrames final: public SampleFrames {
+class WritableSampleFrames final : public SampleFrames {
   public:
     WritableSampleFrames() = default;
     explicit WritableSampleFrames(
             IndexRange frameIndexRange,
             SampleBuffer::WritableSlice writableSlice = SampleBuffer::WritableSlice())
-          : SampleFrames(frameIndexRange),
-            m_writableSlice(writableSlice) {
+            : SampleFrames(frameIndexRange),
+              m_writableSlice(writableSlice) {
```

Comment:
There are many changes like shown below. I am OK with it. I just want to pint it out explicit and verify if the readability is improved. IMHO it is. 

new:
```
    explicit ReadableSampleFrames(
            IndexRange frameIndexRange,
            SampleBuffer::ReadableSlice readableSlice = SampleBuffer::ReadableSlice()
            arg1,
            arg2)
            : SampleFrames(frameIndexRange),
              m_readableSlice(readableSlice), 
              m_init1(1), 
              m_init2(2) {
        statement1();
        statement2();
        statement3();
    }
```

old:
```
    explicit ReadableSampleFrames(IndexRange frameIndexRange,
            SampleBuffer::ReadableSlice readableSlice = SampleBuffer::ReadableSlice(), arg1, arg2)
       : SampleFrames(frameIndexRange),
         m_readableSlice(readableSlice), 
         m_init1(1), 
         m_init2(2) {
        statement1();
        statement2();
        statement3();
    }
```

Reply:
The old indentation was wrong. It must be 8 characters for the ':' (like for functions) and the following initialization statements are aligned vertically.

Splitting of arguments is variable and not enforced by the formatter (see below).

Reply:
Can you verify that the clang-format does it without manual addition?
Than it is fine for me.

Is you fist commit really pure automatic? That will be mandatory for mass reformating in the future.

Reply:
The first commit contains the automatic changes. I will revert the remaining commits, doesn't matter.